notebooks/MAGICC7.ipynb

```diff
@@ -375,9 +646,180 @@
+      "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.1/2.1 MB\u001b[0m \u001b[31m8.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0ma \u001b[36m0:00:01\u001b[0m\n",
+      "\u001b[?25hDownloading SQLAlchemy_Utils-0.41.2-py3-none-any.whl (93 kB)\n",
+      "Downloading toml-0.10.2-py2.py3-none-any.whl (16 kB)\n",
+      "Downloading typer-0.15.2-py3-none-any.whl (45 kB)\n",
+      "Downloading et_xmlfile-2.0.0-py3-none-any.whl (18 kB)\n",
+      "Downloading annotated_types-0.7.0-py3-none-any.whl (13 kB)\n",
+      "Using cached click-8.1.8-py3-none-any.whl (98 kB)\n",
+      "Using cached markdown_it_py-3.0.0-py3-none-any.whl (87 kB)\n",
+      "Downloading mypy-1.15.0-cp313-cp313-macosx_11_0_arm64.whl (9.8 MB)\n",
+      "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m9.8/9.8 MB\u001b[0m \u001b[31m8.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0ma \u001b[36m0:00:01\u001b[0m\n",
+      "\u001b[?25hDownloading shellingham-1.5.4-py2.py3-none-any.whl (9.8 kB)\n",
+      "Downloading starlette-0.46.1-py3-none-any.whl (71 kB)\n",
+      "Downloading typing_inspect-0.9.0-py3-none-any.whl (8.8 kB)\n",
+      "Downloading Mako-1.3.9-py3-none-any.whl (78 kB)\n",
+      "Downloading typeguard-4.4.2-py3-none-any.whl (35 kB)\n",
+      "Using cached mdurl-0.1.2-py3-none-any.whl (10.0 kB)\n",
+      "Downloading mypy_extensions-1.0.0-py3-none-any.whl (4.7 kB)\n",
+      "Installing collected packages: XlsxWriter, wquantiles, typeguard, toml, SQLAlchemy, shellingham, python-dotenv, PyJWT, pydantic-core, psycopg-binary, psycopg, mypy_extensions, mdurl, Mako, et-xmlfile, click, annotated-types, typing_inspect, starlette, SQLAlchemy-Utils, pydantic, openpyxl, mypy, markdown-it-py, alembic, rich, pydantic-settings, pandera, iam-units, fastapi, typer, ixmp4, pyam-iamc\n",
+      "Successfully installed Mako-1.3.9 PyJWT-2.10.1 SQLAlchemy-2.0.39 SQLAlchemy-Utils-0.41.2 XlsxWriter-3.2.2 alembic-1.15.1 annotated-types-0.7.0 click-8.1.8 et-xmlfile-2.0.0 fastapi-0.115.11 iam-units-2023.9.12 ixmp4-0.9.8 markdown-it-py-3.0.0 mdurl-0.1.2 mypy-1.15.0 mypy_extensions-1.0.0 openpyxl-3.1.5 pandera-0.23.1 psycopg-3.2.6 psycopg-binary-3.2.6 pyam-iamc-3.0.0 pydantic-2.10.6 pydantic-core-2.27.2 pydantic-settings-2.8.1 python-dotenv-1.0.1 rich-13.9.4 shellingham-1.5.4 starlette-0.46.1 toml-0.10.2 typeguard-4.4.2 typer-0.15.2 typing_inspect-0.9.0 wquantiles-0.6\n"
+     ]
+    }
+   ],
+   "source": [
+    "# the following packages are required for the plots below\n",
+    "# !pip install pyam-iamc pycollocation sympy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ImportError",
+     "evalue": "cannot import name 'reshape_line_plot' from 'pyam.plotting' (/Users/seangtkelley/miniconda3/envs/climate-model-pymagicc/lib/python3.10/site-packages/pyam/plotting.py)",
```

Comment:
I'm not sure what version of `pyam-iamc` was used in the original run of the notebook, but this function no longer exists and I can't find an equivalent.